csrankings-n.csv

```diff
@@ -28,6 +28,7 @@ Nadarajah Mithulananthan,University of Queensland,http://researchers.uq.edu.au/r
 Nadarajen Veerapen,CRIStAL,https://www.cristal.univ-lille.fr/profil/veerapen,N-1KB5sAAAAJ
 Nadav Amit,Technion,https://nadav.amit.zone/,SxHdnHcAAAAJ
 Nadav Cohen,Tel Aviv University,http://www.cohennadav.com,DmzoCRMAAAAJ
+Nadav Dym,Technion,https://nadavdym.github.io/,qOyXmMYAAAAJ

```

Comment:
Not in CS.

Reply:
Nadav Dym is a researcher specializing in the theory behind deep learning. He publishes mainly in CS conferences such as AAAI, ICML, ICLR, ICCV and NeurIPS (for example, he has 4 papers only at ICML 2024, and one more in AAAI 2024).